fireblocks_sdk/sdk.py

```diff
@@ -1686,6 +1683,16 @@ def get_off_exchanges(self):
 
         return self._get_request(url)
 
+    def get_audit_logs(self, time_period: TimePeriod):
+        """
+        Get audit logs
+        :param time_period: The last time period to fetch audit logs
+        """
+
+        url = f"/v1/audits"
+
+        return self._get_request(url, query_params={'timePeriod': time_period})
```

Comment:
Is the timePeriod query param really in camelCase? 🤦🏻‍♂️

Reply:
That's right